lib/widgets/action_sheet.dart

```diff
@@ -247,30 +248,56 @@ class ReactionButtons extends StatelessWidget {
     }
 
     return Container(
-      padding: const EdgeInsets.all(8),
+      padding: const EdgeInsets.only(left: 8),
       decoration: BoxDecoration(color: designVariables.contextMenuItemBg.withFadedAlpha(0.12)),
-      child: Row(
-        mainAxisAlignment: MainAxisAlignment.spaceAround,
-        children: List.unmodifiable(popularEmojis.map((emoji) {
-          final selfVoted = hasSelfVote(emoji);
-          return IconButton(
-            onPressed: () => _onPressed(emoji, selfVoted, zulipLocalizations),
-            isSelected: selfVoted,
-            style: IconButton.styleFrom(
-              padding: EdgeInsets.zero,
-              splashFactory: NoSplash.splashFactory,
-              shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(3.5)),
-              tapTargetSize: MaterialTapTargetSize.shrinkWrap,
-              visualDensity: VisualDensity.compact,
-            ).copyWith(backgroundColor: WidgetStateColor.resolveWith((states) =>
-              states.any((e) => e == WidgetState.pressed || e == WidgetState.selected)
-                ? designVariables.contextMenuItemBg.withFadedAlpha(0.20)
-                : Colors.transparent)),
-            icon: UnicodeEmojiWidget(
-              emojiDisplay: emoji.emojiDisplay as UnicodeEmojiDisplay,
-              notoColorEmojiTextSize: 20.1,
-              size: 24));
-        })))
+      child: Row(children: [
+        Flexible(child: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 8),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: List.unmodifiable(popularEmojis.map((emoji) {
+              final selfVoted = hasSelfVote(emoji);
+              return IconButton(
+                onPressed: () => _onPressed(emoji, selfVoted, zulipLocalizations),
+                isSelected: selfVoted,
+                style: IconButton.styleFrom(
+                  padding: EdgeInsets.zero,
+                  splashFactory: NoSplash.splashFactory,
+                  shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(3.5)),
+                  tapTargetSize: MaterialTapTargetSize.shrinkWrap,
+                  visualDensity: VisualDensity.compact,
+                ).copyWith(backgroundColor: WidgetStateColor.resolveWith((states) =>
+                  states.any((e) => e == WidgetState.pressed || e == WidgetState.selected)
+                    ? designVariables.contextMenuItemBg.withFadedAlpha(0.20)
+                    : Colors.transparent)),
+                icon: UnicodeEmojiWidget(
+                  emojiDisplay: emoji.emojiDisplay as UnicodeEmojiDisplay,
+                  notoColorEmojiTextSize: 20.1,
+                  size: 24));
+            }))))),
+        TextButton(
```

Comment:
I think `TextButton` is adding 2px padding at the top and 2px at the bottom, to make the button's height 48px instead of 44px as in the Figma. It looks like we don't use `TextButton` in the emoji autocomplete. What if we followed that example?